packages/fx-core/src/question/options/CreateProjectOptions.ts

```diff
@@ -53,6 +53,11 @@ export const CreateProjectOptions: CLICommandOption[] = [
       "BotAndMessageExtension",
       "TabNonSsoAndBot",
       "taskpane",
+      "sso",
+      "react",
+      "cfShared",
+      "cfJS",
```

Comment:
Why use "cfShared" instead of "CustomFunctionShared"?

Reply:
CLI change Maybe Reafctor in the future.